localization/english/CWP/africa/CWP_libya_l_english.yml

```
@@ -0,0 +1,140 @@
+
+  #Events
+  libya.1.t: "The Mad Dog of the Middle East"
+  libya.1.d: "In the midst of political turbulence and simmering discontent, Colonel Muammar Gaddafi launches a daring coup attempt to seize control of Libya. Fueled by his unyielding ambition, Gaddafi aspires to rewrite the course of history and establish himself as the undisputed ruler of the nation."
+  libya.1.f: "I am an international leader, the dean of the Arab rulers, the king of kings of Africa and the imam of Muslims, and my international status does not allow me to descend to a lower level.\n― Muammar Gaddafi"
+  libya.1.a: "Form a transitional government"
+  libya.1.b: "Proclaim a socialist state"
+  libya.1.c: "The revolution fails"
+
+  libya.3.t: "Gaddafis Dream"
+  libya.3.d: "In a groundbreaking address to the nation, Libyan leader Muammar Gaddafi announces a momentous decision to centralize and fortify his diverse states into a cohesive and formidable union. With unwavering conviction, Gaddafi sets forth a grand vision that transcends borders, cultures, and historical divisions."
+  libya.3.f: "I am an international leader, the dean of the Arab rulers, the king of kings of Africa and the imam of Muslims, and my international status does not allow me to descend to a lower level.\n― Muammar Gaddafi"
+  libya.3.a: "Gaddafis dream is our dream"
+  libya.3.b: "This is madness"
+
+  libya.4.t: "Gaddafis Influence spreads"
+  libya.4.d: "In the face of diplomatic pressures, our nation has chosen a path of unity and cooperation, joining hands with Libya to form a new and powerful alliance. Faced with challenges that tested our resolve, we realized that standing alone might not be enough to confront the complexities of the modern world. Thus, we embraced the vision of a stronger future by collaborating with our Libyan counterparts."
+  libya.4.f: "There is no state with a democracy except Libya on the whole planet.\n― Muammar Gaddafi"
+  libya.4.a: "And now?"
+  libya.4.tt: "If Libya tries to annex us, we can declare our independence by force"
+
+  libya.5.t: "Libyan Ambitions"
+  libya.5.d: "Muammar Gaddafi, the leader of Libya, sets his sights on expanding his influence far beyond his nation's borders. With unwavering determination, Gaddafi seeks to bring our country into his fold, forming an unprecedented union that could potentially reshape the geopolitical landscape."
+  libya.5.f: "When I met Nasser, he said to me, 'I see myself when I was young in you. You are the future for the Arab revolution.' This meant very, very much to me.\n― Muammar Gaddafi"
+  libya.5.a: "Are we in danger?"
+  libya.5.tt: "A new journal entry becomes available"
+
+  libya.6.t: "[SCOPE.sCountry('gaddafi_unification_target_scope').GetName] joins our Union"
```

- capitals in titles
```suggestion
  libya.6.t: "[SCOPE.sCountry('gaddafi_unification_target_scope').GetName] Joins Our Union"
```